mitosheet/mitosheet/mito_config.py

```diff
@@ -5,65 +5,89 @@
 # }
-mec_upgrade_functions: Dict[int, Any] = {}	
+mec_upgrade_functions: Dict[int, Any] = {}
 
 def upgrade_mito_enterprise_configuration(mec: Optional[Dict[str, Any]]) -> Optional[Dict[str, Any]]:
     if mec is None:
         return None
 
     # So mypy tests recognize that mec is not None
     _mec = mec
-    while _mec[MEC_CONFIG_KEY_VERSION] in mec_upgrade_functions:
-        _mec = mec_upgrade_functions[_mec[MEC_CONFIG_KEY_VERSION]](_mec)
+    while _mec[MITO_CONFIG_VERSION_KEY] in mec_upgrade_functions:
+        _mec = mec_upgrade_functions[_mec[MITO_CONFIG_VERSION_KEY]](_mec)
 
     return _mec
 
+# Since Mito needs to look up indivdual environment variables, we need to 
+# know the names of the variables associated with each mito config version. 
+# To do so we store them as a list here. 
+mec_version_keys = {
+    '1': [MITO_CONFIG_VERSION_KEY, MITO_CONFIG_SUPPORT_EMAIL_KEY]
+}
+
+def create_mec_from_environment_variables() -> Optional[Dict[str, Any]]:
+    """
+    Creates a Mito Enterprise Config object from the environment variables
+    """
+    config_version = os.environ.get(MITO_CONFIG_VERSION_KEY)
+
+    if config_version is None:
+        return None
+
+    mec: Dict[str, Any] = {}
+    for key in mec_version_keys[config_version]:
+        mec[key] = os.environ.get(key)
+
+    return mec
+
 class MitoConfig:
     """
-    The MitoConfig class is repsonsible for 1) reading the settings from the 
-    mito_helper_config package if it is installed and 2) returning the configuration variables
-    used by the rest of the app. 
+    The MitoConfig class is repsonsible for reading the settings from the 
+    environment variables and returning them as the most updated version of the 
+    mito_enterprise_configuration object that is used by the rest of the app. 
 
-    If the mito_helper_configuration package does not exist or does not set every configuration option, 
+    If the environment variables does not exist or does not set every configuration option, 
     the MitoConfig class sets defaults. 
-
-    See https://github.com/mito-ds/mitosheet_helper_config/blob/main/mitosheet_helper_config/README.md for
-    information about developing with the mito_helper_config package.
     """
-    def __init__(self, mito_enterprise_configuration: Optional[Dict[str, Any]]):
-        self.mec = upgrade_mito_enterprise_configuration(mito_enterprise_configuration)
+    def __init__(self):
+        mec_potentially_outdated = create_mec_from_environment_variables()
+        self.mec = upgrade_mito_enterprise_configuration(mec_potentially_outdated)
 
-    def get_version(self) -> Optional[int] :
-        if self.mec is None:
-            return 1 # NOTE: update this to be the most recent version, when we bump the version
-        return self.mec[MEC_CONFIG_KEY_VERSION]
+    def get_version(self) -> str:
+        if self.mec is None or self.mec[MITO_CONFIG_VERSION_KEY] is None:
+            log('loaded_mito_enterprise_config')
```

Comment:
I think that this log should be in the init, if the `self.mec is not None`

That way, we can make sure we always log when we create it, even if we somehow move to not using this get_version thing.